content/applications/websites/livechat.rst

```diff
@@ -5,80 +5,287 @@
+selection.
+
+.. tip::
+   Color selection, for the button or header, can be made manually, or through RGB, HSL, or HEX code
+   selection from the pop-up color selection window that appears when those color bubbles are
+   selected. Different options will be available, depending on your operating system.
+
+Livechat window
+~~~~~~~~~~~~~~~
+
+The *Livechat Window* is the where the live chat conversation takes place.
+
+Edit the :guilabel:`Welcome Message` to change the message a visitor sees when they open the chat
+window. This message will appear as though it is sent by a live chat operator, and should be an
+invitation to continue the conversation.
+
+Edit the :guilabel:`Chat Input Placeholder` to alter the text that appears in the box where visitors
+will type their replies.
+
+The *Channel Header* is the colored bar at the top of the chat window. The :guilabel:`Channel Header
+color` can be changed following the same steps as the :guilabel:`Livechat Button Color` above.
+
+.. figure:: livechat/chat-window.png
+   :align: center
+
+   The Livechat Window with a purple header. The chat input placeholder reads "Ask Something..."
+
+Channel rules
+-------------
+
+The *Channel Rules* tab on the live chat channel details form determines when the *Live Chat Window*
+opens on the website, by configuring when a *URL Regex* action is triggered (e.g., a page visit).
+
+To create a new channel rule, click :guilabel:`Add a line`. This opens the :guilabel:`Open: Rules`
+pop-up.
+
+.. image:: livechat/create-rules.png
+   :align: center
+   :alt: View of a channel's rules form for Odoo Live Chat.
+
+Create new rules
+~~~~~~~~~~~~~~~~
+
+Fill out the fields on the :guilabel:`Open: Rules` pop-up as instructed below, then click
+:guilabel:`Save`.
+
+.. tabs::
+
+   .. tab:: Live Chat Button
 
-External options
-----------------
+      The *Livechat Button* is the icon that appears in the bottom corner of the website. Select
+      from one of the following display options:
 
-| If your website was not created with Odoo, you can find the code to be added to your own, under
-  the *Widget* tab.
-| Odoo also offers an URL you can send to users so they can have access to a single live chat page.
+      - :guilabel:`Show` displays the chat button on the page(s).
+      - :guilabel:`Show with notification` displays the chat button, as well as a floating text
+        bubble next to the button.
+      - :guilabel:`Open automatically` displays the button and automatically opens the chat window
+        after a specified amount of time (designated in the :guilabel:`Open automatically timer`
+        field).
+      - :guilabel:`Hide` hides the chat button on the page(s).
 
-.. image:: livechat/widget.png
-   :alt: View of the widget tab for Odoo Live Chat
+   .. tab:: Chatbot
 
-Managing chat requests
-======================
+      If a :guilabel:`Chatbot` will be included on this channel, select it from the dropdown. If the
```

Comment:
I feel the guilabel on chatbot here is unecessary since we are talking about a chatbot in general